src/test/TestGsm.t.sol

```diff
@@ -1330,9 +1332,6 @@ contract TestGsm is TestGhoBase {
     uint256 seizedAmount = GHO_GSM.seize();
     assertEq(seizedAmount, DEFAULT_GSM_USDC_AMOUNT, 'Unexpected seized amount');
 
-    vm.expectRevert('FACILITATOR_BUCKET_LEVEL_NOT_ZERO');
-    GHO_TOKEN.removeFacilitator(address(GHO_GSM));
```

Comment:
This was to validate that we cannot offboard the facilitator yet. The analogous would be an assert that usedGho > 0